homeassistant/components/geniushub/sensor.py

```diff
@@ -58,6 +61,32 @@ def name(self):
         """Return the name of the sensor."""
         return self._name
 
+    @property
+    def icon(self):
+        """Return the icon of the sensor."""
+        values = self._device._info_raw['childValues']  # noqa; pylint: disable=protected-access
```

Comment:
It would be nice if the api can be changed so we don't need to access private attributes.

Reply:
This is planned for (I am the author of the api) - the problems is that the vendor has plans to change their public api, and I'm waiting for that...